backend/server.go

```diff
@@ -24,30 +22,25 @@ func main() {
 	router := chi.NewRouter()
 
 	// Add CORS middleware around every request
-	// See https://github.com/rs/cors for full option listing
 	router.Use(cors.New(cors.Options{
 		AllowedOrigins:   []string{"http://localhost:5173", "http://localhost:8080", "https://studygator.chasemacdonnell.net"},
 		AllowCredentials: true,
 		Debug:            true,
 	}).Handler)
 
+	// Serve static files from the images directory
+	fs := http.FileServer(http.Dir("./graph/images"))
+	router.Handle("/images/*", http.StripPrefix("/images/", fs))
```

Comment:
Shouldn't really be serving static files for resources like this, should use an object storage/uploads library for dynamic data and static data should always be on the frontend.